### PR TITLE
#75 Game which can draw (with stylecop)

### PR DESCRIPTION
**Pull request by MilRoad**
#75 Game which can draw (with stylecop)

### CODE REVIEW

File: HW3/Milena Laistseva/GameWhichCanDraw/GameWhichCanDraw/DrawOnConsoleBoard.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            {
+                radius = board.boardSizeY / 4;
+            }
+
+            int x0 = board.boardSizeX / 2;
+            int y0 = board.boardSizeY / 2;
+
+            for (int alpha = 0; alpha < 360; alpha ++)
+            {
+                board.WriteAt("*", Convert.ToInt32(radius * Math.Cos(alpha))+ x0, Convert.ToInt32(radius * Math.Sin(alpha)) + y0);
```
Comment (vladimirchr) [inline HW3/Milena Laistseva/GameWhichCanDraw/GameWhichCanDraw/DrawOnConsoleBoard.cs]: Great idea and great implementation)

File: HW3/Milena Laistseva/GameWhichCanDraw/GameWhichCanDraw/Game.cs [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                else
+                {
+                    ioDevice.WriteOutput(phraseProvider.GetPhrase("ItIsNotANumber"));
+                }
+
+            } while (result < 0);
+
+            if (result == 0)
+            {
+                enteredNumber = -1;
```
Comment (vladimirchr) [inline HW3/Milena Laistseva/GameWhichCanDraw/GameWhichCanDraw/Game.cs]: for me it was not easy to figure it out how input processing)

File: HW3/Milena Laistseva/GameWhichCanDraw/GameWhichCanDraw/DrawOnConsoleBoard.cs (hunk 1 above)
Comment (MilRoad) [reply HW3/Milena Laistseva/GameWhichCanDraw/GameWhichCanDraw/DrawOnConsoleBoard.cs]: Thanks :)

File: HW3/Milena Laistseva/GameWhichCanDraw/GameWhichCanDraw/Game.cs (hunk 2 above)
Comment (MilRoad) [reply HW3/Milena Laistseva/GameWhichCanDraw/GameWhichCanDraw/Game.cs]: Agree, I'll add some comments)

File: HW3/Milena Laistseva/GameWhichCanDraw/GameWhichCanDraw/JsonPhraseProvider.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    {
+        private readonly string language;
+
+        public JsonPhraseProvider(string language)
+        {
+            this.language = language;
+        }
+        public string GetPhrase(string phraseKey)
+        {
+            var resourceFile = new FileInfo($"..\\..\\Resources\\Lang{language}.json");
```
[the remark this answers is not in the document]
Comment (MilRoad) [reply HW3/Milena Laistseva/GameWhichCanDraw/GameWhichCanDraw/JsonPhraseProvider.cs]: Thanks! It's better :)
But I think I'll do that next time.

**Pull request merged**